packages/wicked/1004-adjust-safeguard-for-dhcp6-defer-timeout.patch

```diff
@@ -13,18 +13,18 @@ Signed-off-by: Ben Cressey <bcressey@amazon.com>
  1 file changed, 1 insertion(+), 1 deletion(-)
 
 diff --git a/src/dhcp6/fsm.c b/src/dhcp6/fsm.c
-index b3284f6..690e365 100644
+index 08624b49..b864efd6 100644
 --- a/src/dhcp6/fsm.c
 +++ b/src/dhcp6/fsm.c
-@@ -563,7 +563,7 @@ ni_dhcp6_fsm_timeout(ni_dhcp6_device_t *dev)
+@@ -569,7 +569,7 @@ ni_dhcp6_fsm_timeout(ni_dhcp6_device_t *dev)
  			/* Do we still need this safeguard? */
- 			deadline = ni_dhcp6_remaining_time(&dev->start_time,
- 						dev->config->defer_timeout);
+ 			deadline = ni_lifetime_left(dev->config->defer_timeout,
+ 					&dev->start_time, NULL);
 -			if (deadline) {
 +			if (deadline > 1) {
- 				deadline *= 1000;
- 				ni_dhcp6_fsm_set_timeout_msec(dev, deadline);
+ 				ni_dhcp6_fsm_set_timeout_sec(dev, deadline);
```

Comment:
Do we still observe roughly 1 second but not much longer spent waiting for a DHCPv6 lease if DHCPv6 isn't configured for the instance in EC2?

Reply:
```
Feb 03 18:51:06.518909 localhost wickedd-dhcp6[1453]: eth0: Request to acquire DHCPv6 lease with UUID 1a24fc61-1cdd-0700-be05-000003000000 in mode auto
...
Feb 03 18:51:07.518884 localhost wickedd-dhcp6[1453]: eth0: timeout in state INIT
```

This seems right to me!